sensor_msgs/Image rgb_img
sensor_msgs/Image mask
---
int64 result

